public/planning/postes_cfg/ajax.infos.php

```diff
@@ -25,7 +25,7 @@
 $id=filter_input(INPUT_GET, "id", FILTER_SANITIZE_NUMBER_INT);
 $CSRFToken = filter_input(INPUT_GET, "CSRFToken", FILTER_SANITIZE_STRING);
 $nombre=filter_input(INPUT_GET, "nombre", FILTER_SANITIZE_NUMBER_INT);
-$nom=filter_input(INPUT_GET, "nom", FILTER_SANITIZE_STRING);
```

Comment:
why this change ? I'm not sure this file is used in the framework index page

Reply:
As far as I can remember, it was a matter of display while I as testing the whole. I will set this aside for now.

Reply:
> As far as I can remember, it was a matter of display while I as testing the whole. I will set this aside for now.

I confirm my explanation : with the filter, quotes cannot be added.